projects/metis/src/app/app.component.ts

```diff
@@ -113,13 +113,14 @@ export class AppComponent extends SubscriptionManager implements OnInit {
   */
   cancelWorkflow(): void {
     if (this.cancellationRequest) {
+      this.errorNotification = undefined;
       this.subs.push(
         this.workflows.cancelThisWorkflow(this.cancellationRequest.workflowExecutionId).subscribe(
           () => {
-            console.log('cancelling');
+            // successful cancellation request made
```

Comment:
In general logs shouldn't be there unless debugging.  I think this statement was only there to stop the linter complaining about an empty function, but now it doesn't complain because of the comment.